### PR TITLE
Chapter 9

### PR DESCRIPTION
**Pull request by mohdzakwan99**

### CODE REVIEW

**Comment by mohdzakwan99** [comment]
ready

File: test/controllers/users_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test "should get show" do
```
Comment (ngocvt-0484) [inline test/controllers/users_controller_test.rb:4]: dont need do test
please delete this file

File: package.json [hunk 2]
```diff
@@ -6,8 +6,11 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.4.2",
+<<<<<<< HEAD
```
Comment (ngocvt-0484) [inline package.json:9]: please fix conflict on this file

**Comment by mohdzakwan99** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged
You can merge this pull request

**Pull request merged**